cnf-certification-test/accesscontrol/securitycontextcontainer/securitycontextcontainer.go

```diff
@@ -0,0 +1,454 @@
+		result = false
+	}
+	if refCategory.HaveDropCapabilities == containerSCC.HaveDropCapabilities {
+		tnf.ClaimFilePrintf("DropCapabilities list - OK")
+	} else {
+		tnf.ClaimFilePrintf("HaveDropCapabilities = %s but expected  %s - NOK", containerSCC.HaveDropCapabilities, refCategory.HaveDropCapabilities)
+		tnf.ClaimFilePrintf("its didnt have all the required (MKNOD, SETUID, SETGID, KILL)/(ALL) drop value ")
+		result = false
+	}
+	if refCategory.HostDirVolumePluginPresent == containerSCC.HostDirVolumePluginPresent {
+		tnf.ClaimFilePrintf("HostDirVolumePluginPresent = %s - OK", containerSCC.HostDirVolumePluginPresent)
+	} else {
+		tnf.ClaimFilePrintf("HostDirVolumePluginPresent = %s but expected  %s - NOK", containerSCC.HostDirVolumePluginPresent, refCategory.HostDirVolumePluginPresent)
+		result = false
+	}
+	if refCategory.HostIPC >= containerSCC.HostIPC {
+		tnf.ClaimFilePrintf("HostIPC = %s - OK", containerSCC.HostIPC)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("HostIPC = %s but expected <= %s - NOK", containerSCC.HostIPC, refCategory.HostIPC)
+	}
+	if refCategory.HostNetwork >= containerSCC.HostNetwork {
+		tnf.ClaimFilePrintf("HostNetwork = %s - OK", containerSCC.HostNetwork)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("HostNetwork = %s but expected <= %s - NOK", containerSCC.HostNetwork, refCategory.HostNetwork)
+	}
+	if refCategory.HostPID >= containerSCC.HostPID {
+		tnf.ClaimFilePrintf("HostPID = %s - OK", containerSCC.HostPID)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("HostPID = %s but expected <= %s - NOK", containerSCC.HostPID, refCategory.HostPID)
+	}
+	if refCategory.HostPorts >= containerSCC.HostPorts {
+		tnf.ClaimFilePrintf("HostPorts = %s - OK", containerSCC.HostPorts)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("HostPorts = %s but expected <= %s - NOK", containerSCC.HostPorts, refCategory.HostPorts)
+	}
+	if refCategory.PrivilegeEscalation >= containerSCC.PrivilegeEscalation {
+		tnf.ClaimFilePrintf("HostNetwork = %s - OK", containerSCC.HostNetwork)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("PrivilegeEscalation = %s but expected <= %s - NOK", containerSCC.PrivilegeEscalation, refCategory.PrivilegeEscalation)
+	}
+	if refCategory.PrivilegedContainer >= containerSCC.PrivilegedContainer {
+		tnf.ClaimFilePrintf("PrivilegedContainer = %s - OK", containerSCC.PrivilegedContainer)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("PrivilegedContainer = %s but expected <= %s - NOK", containerSCC.PrivilegedContainer, refCategory.PrivilegedContainer)
+	}
+	if refCategory.ReadOnlyRootFilesystem >= containerSCC.ReadOnlyRootFilesystem {
+		tnf.ClaimFilePrintf("ReadOnlyRootFilesystem = %s - OK", containerSCC.ReadOnlyRootFilesystem)
+	} else {
+		result = false
+		tnf.ClaimFilePrintf("ReadOnlyRootFilesystem = %s but expected <= %s - NOK", containerSCC.ReadOnlyRootFilesystem, refCategory.ReadOnlyRootFilesystem)
+	}
+	if refCategory.SeLinuxContextPresent == containerSCC.SeLinuxContextPresent {
```

Comment:
Add comment:
// SeLinuxContextPresent is true if the SeLinuxContext field is present and set to a value (e.g. not nil), false otherwise.